core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/ShardingContainerPoolBalancer.scala

```diff
@@ -302,9 +290,9 @@ class ShardingContainerPoolBalancer(
     chosen
       .map { invoker =>
         val memoryLimit = action.limits.memory
-        val memoryLimitInfo = if (memoryLimit == stdActionMemoryLimit) { "std" } else { "non-std" }
+        val memoryLimitInfo = if (memoryLimit == MemoryLimit.defaultMemoryLimit) { "std" } else { "non-std" }
```

Comment:
With your new implementation

```
  /** Gets MemoryLimit with default value */
  protected[core] def apply(): MemoryLimit = MemoryLimit(stdMemory)	  protected[core] def apply(): MemoryLimit = defaultMemoryLimit
```

shouldn't using `MemoryLimit()` be fast enough here? It will return the new singleton...